### PR TITLE
FWF:3557 and 3603 [feature] Form create and mapper create and auth create apis combined into form-design

### PR DESCRIPTION
**Pull request by shuhaib-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes

# Screenshots (if applicable)

# Notes

# Checklist
- [ ] Updated changelog
- [ ] Added meaningful title for pull request

### CODE REVIEW

**Comment by shuhaib-aot** [comment]
Pylint fixed in this pr : https://github.com/AOT-Technologies/forms-flow-ai/pull/2236/

File: forms-flow-api/src/formsflow_api/services/form_process_mapper.py [hunk 1]
```diff
@@ -286,7 +290,52 @@ def form_design_update(data, form_id):
...
+        response = formio_service.create_form(data, form_io_token)
+        form_id = response.get("_id")
+        # create default data form mapper table
+        mapper_data = {
+            "formId": form_id,
+            "formName": response.get("title"),
+            "description": data.get("description", ""),
+            "formType": response.get("type"),
+            "processKey": response.get("name"),
+            "processName": response.get("name"),
```
Comment (arun-s-aot) [inline forms-flow-api/src/formsflow_api/services/form_process_mapper.py:314]: is it correct that we are setting processKey and processName from 'name' ? @shuhaib-aot

Comment (auslin-aot) [reply forms-flow-api/src/formsflow_api/services/form_process_mapper.py:314]: form name is used as process_name & process_key

File: forms-flow-api/src/formsflow_api/resources/form_process_mapper.py [hunk 2]
```diff
@@ -407,23 +386,30 @@ def delete(mapper_id: int):
     )
     def put(mapper_id: int):
         """Update form by mapper_id."""
-        application_json = request.get_json()
-
-        if "taskVariable" in application_json:
+        data = request.get_json()
+        mapper_data = data.get("mapperData")
```
Comment (sumesh-aot) [inline forms-flow-api/src/formsflow_api/resources/form_process_mapper.py:390]: Can we change the keys to match resources ?
- mapper
- authorizations
- taskVariables

Comment (shuhaib-aot) [reply forms-flow-api/src/formsflow_api/resources/form_process_mapper.py:390]: okay

**Pull request closed without merging**